controls/roles/manage-service/molecule/monitor-ssvnetwork/prepare.yml

```diff
@@ -93,7 +93,7 @@
 
     - name: Create service config directory
       file:
-        path: /opt/app/services/{{ ssv_service }}/data/blox/ssv/
+        path: /opt/app/services/{{ ssv_service }}/data/ssv/network/
```

Comment:
There's actually no need to `ssv/network` in the path, it's already in the ssv.network's data path.

My suggestion: `/opt/app/services/{{ ssv_service }}/data`